src/metabase/cmd/copy.clj

```diff
@@ -345,7 +311,27 @@
 
 (def ^:private entities-without-autoinc-ids
   "Entities that do NOT use an auto incrementing ID column."
-  #{Setting Session DataMigrations ImplicitAction HTTPAction QueryAction})
+  #{:model/Setting
+    :model/Session
+    :model/DataMigrations
+    :model/ImplicitAction
+    :model/HTTPAction
+    :model/QueryAction
+    :model/ModelIndexValue
+    :model/TablePrivileges})
+
+(defmulti ^:private postgres-id-sequence-name
+  {:arglists '([model])}
+  keyword)
+
+(defmethod postgres-id-sequence-name :default
+  [model]
+  (str (name (t2/table-name model)) "_id_seq"))
+
+;;; we changed the table name to `sandboxes` but never updated the underlying ID sequences or constraint names.
+(defmethod postgres-id-sequence-name :model/GroupTableAccessPolicy
+  [_model]
+  "group_table_access_policy_id_seq")
```

Comment:
Does this really have to be extensible? Wouldn't it be simpler to use a `case` form?